README.md

```diff
@@ -1,11 +1,14 @@
 > FLEDGE has been renamed to Protected Audience API. To learn more about the name change, see the [blog post](https://privacysandbox.com/intl/en_us/news/protected-audience-api-our-new-name-for-fledge)
 
-# Protected Audience documentation
+# Protected Audience Services documentation
 
 This GitHub repository contains proposals for the Privacy Sandbox's
 Protected Audience server-side infrastructure that supports the Privacy
 Sandbox's client-side APIs.
 
+> [!IMPORTANT]
+> This repository is for documentation only. For discussions, visit the [WICG/protected-auction-services-discussion](https://github.com/WICG/protected-auction-services-discussion) repository.
```

Comment:
Link is broken. Or do you intend to rename that repo?

Reply:
NM I meant to comment on a broken link in the new repo pointing back to here.

Reply:
Hi @dmdabbs! We just sent out an announcement for renaming the repository on Tuesday.  Take a look at the post here: https://groups.google.com/a/chromium.org/g/fledge-api-announce/c/2kErrNQH8yM